go/libkb/gpg_cli.go

```diff
@@ -170,6 +170,35 @@ func (g *GpgCLI) ExportKey(k PGPKeyBundle) (err error) {
 	return PickFirstError(e1, e2, e3)
 }
 
+func (g *GpgCLI) Sign(fp PGPFingerprint, payload []byte) (string, error) {
+	arg := RunGpg2Arg{
+		Arguments: []string{"--armor", "--sign", "--default-key", fp.String()},
```

Comment:
They seem to be the same:

    -u, --local-user name
    Use name as the user ID to sign. This option is silently ignored for the list commands, so that it can be used in an options file.

    --default-key name
    Use name as default user ID for signatures. If this is not used the default user ID is the first user ID found in the secret keyring.